mobile-save-for-later/conf/cfn.yaml

```diff
@@ -1,48 +1,4 @@
 Resources:
-  SaveForLaterWriteThrottleEvents:
-    Type: 'AWS::CloudWatch::Alarm'
-    Properties:
-      EvaluationPeriods: 1
-      TreatMissingData: notBreaching
-      Dimensions:
-        - Name: TableName
-          Value: !Ref SaveForLaterDynamoTable
-      AlarmActions:
-        - !Ref DynamoNotificationTopic
-      Namespace: AWS/DynamoDB
-      Period: 60
-      ComparisonOperator: GreaterThanThreshold
-      AlarmName: !Sub '${App}-${Stage}-articles-WriteCapacityUnitsLimit-BasicAlarm'
-      ActionsEnabled: !FindInMap
-        - StageVariables
-        - !Ref Stage
-        - AlarmActionsEnabled
-      Statistic: Sum
-      Threshold: 0
-      Unit: Count
-      MetricName: WriteThrottleEvents
-  SaveForLaterReadThrottleEvents:
-    Type: 'AWS::CloudWatch::Alarm'
-    Properties:
-      EvaluationPeriods: 1
-      TreatMissingData: notBreaching
-      Dimensions:
-        - Name: TableName
-          Value: !Ref SaveForLaterDynamoTable
-      AlarmActions:
-        - !Ref DynamoNotificationTopic
-      Namespace: AWS/DynamoDB
-      Period: 60
-      ComparisonOperator: GreaterThanThreshold
-      AlarmName: !Sub '${App}-${Stage}-articles-ReadCapacityUnitsLimit-BasicAlarm'
-      ActionsEnabled: !FindInMap
-        - StageVariables
-        - !Ref Stage
-        - AlarmActionsEnabled
```

Comment:
Minor: I think you can remove `AlarmActionsEnabled` from `Mappings` now too

Reply:
👍